src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1202,6 +1203,7 @@ void runtime::cpu::CPU_ExternalFunction::register_common_passes(
         return true;
     };
 
+    REGISTER_KNOBBED_PASS(Opset1Downgrade, true, ngraph::pass)
```

Comment:
@aprocter Is it correct place to perform `Opset1Downgrade` pass?

Reply:
I think so! CPU BE folks should double check.